src/common/transformations/tests/op_conversions/convert_interpolate11_downgrade_test.cpp

```diff
@@ -60,6 +60,7 @@ std::shared_ptr<ov::Model> create_v4_model(const bool with_axes,
     attributes.pads_end = {0, 0};
 
     const auto input = std::make_shared<ov::opset11::Parameter>(ov::element::i32, ov::Shape{1, 2, 10, 10});
+    const auto axes = std::make_shared<ov::opset4::Parameter>(ov::element::i32, ov::Shape{2});
```

Comment:
Can we use operations from one opset :)

Reply:
I think we can but this is just a test where I still need to include 2 headers and this form clearly shows that the expected model uses opset4 operations. Should I change them to v11 and leave just opset4::Interpolate?